src/fmgen/psg.cpp

```diff
@@ -118,7 +118,7 @@ void PSG::MakeEnvelopTable()
         2,2, 2,0, 2,1, 2,3, 1,1, 1,3, 1,2, 1,0,
     };
     static uint8 table2[4] = {  0,  0, 31, 31 };
-    static uint8 table3[4] = {  0,  1, -1,  0 };
+    static uint8 table3[4] = {  0,  1, 255,  0 };
```

Comment:
```suggestion
    static uint8 table3[4] = {  0,  1, 255, 0 };
```
For alignment with the previous line.